src/components/Loaders/RadarLoader.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react'
+import { Animated, Easing } from 'react-native'
+import Box from '../Box'
+
+const RadarLoader = ({
+  duration = 2000,
+  color = 'purple',
+}: {
+  duration?: number
+  color?: 'purple' | 'green'
+}) => {
+  const rotateAnim = useRef(new Animated.Value(0))
+
+  const images: Record<string, any> = {
+    purple: require('../../assets/images/radar-loader-purple.png'),
+    green: require('../../assets/images/radar-loader-green.png'),
+  }
+
```

Comment:
that doesn't loop infinitely though does it? I was trying to find a way to do it without defining a duration since we can't always predict how long certain operations would take